mkdocs/topics/deployments.md

```diff
@@ -477,4 +477,8 @@ jobs:
         env:
           VISIVO_TOKEN: ${{ secrets.VISIVO_TOKEN }}
 ```
-{% endraw %}
\ No newline at end of file
+{% endraw %}
+
+## Static
+
+You can also manage and host the dashboard internally if you wish to manage it entirely within your systems.  To create a self-contained build all you need to do is run the `visivo dist` command.  This will create a folder that you can use to host in any manner that suits your needs.
```

Comment:
I don't think this blocks merge, but we could add something about how to deploy it with different providers like netlify too.

Reply:
Netlify doesn't support python 3.10 ... very odd.